Makefile

```diff
@@ -0,0 +1,13 @@
+DC_RUN_PHP = docker-compose run php
```

Comment:
This Makefile will probably make things easier.  We will probably want to update the travis.yaml to use some of these functionalities.